src/styles/utils/veriables.scss

```diff
@@ -0,0 +1,9 @@
+$tablet-min-width: 744px;
+$desktop-min-width: 1260px;
+$page-main-font: Poppins, sans-serif;
+$main-text-color: #fff;
+$secondary-text-color: #1d1d1d;
+$background-color: #1d1d1d;
+$form-color: #F8F8FA;
+$form-text-color: #292929;
+$from-placeholder-color: #7E7E83;
```

Comment:
Describe all in lowercase.

Reply:
It`s a Mate code-style rule. 
Companies have different code styles, but 100% you need to describe the color variables in the same lower/upper/camel case.